plugins/pulp_rpm/plugins/error_codes.py

```diff
@@ -13,5 +13,3 @@
 RPM1004 = Error("RPM1004", _("Error retrieving metadata: %(reason)s"), ['reason'])
 RPM1005 = Error("RPM1005", _("Unable to sync a repository that has no feed."), [])
 RPM1006 = Error("RPM1006", _("Could not parse repository metadata"), [])
-RPM1007 = Error("RPM1007", _("Could not parse errata `updated` field: expected format "
```

Comment:
I believe error codes are designed to be immutable. This error code is already included in a released version of Pulp so we shouldn't remove it now. https://github.com/pulp/pulp_rpm/blob/2.8-release/plugins/pulp_rpm/plugins/error_codes.py#L16
Often times I just leave them there and remove all references to them so that it's just unused.